{{cookiecutter.project_name}}/tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
+envlist = py27, py36, linting, docs
+
+[travis]
+python =
+    3.6: py36
+    3.5: py35
+    2.7: py27
+
+[testenv]
+passenv = TOXENV CI TRAVIS TRAVIS_* APPVEYOR APPVEYOR_*
+extras = testing
+commands =
+    pytest --cov={envsitepackagesdir}/{{ cookiecutter.package_name }}
+    codecov
+
+[testenv:linting]
+skip_install = True
+basepython = python3.6
+deps = pre-commit>=1.11.0
+commands = pre-commit run --all-files --show-diff-on-failure
+
+[testenv:docs]
+skipsdist = True
+usedevelop = True
+changedir = docs
+deps =
```

Comment:
Instead of declaring `deps`, use `extras = docs` instead. 😉

Reply:
✔️  Done